carlotEvents.js

```diff
@@ -0,0 +1,44 @@
+// function activateEvents() {
+
+var carLot = (function(eventHandlers) {
+
+
+
+    var input = document.getElementById("descriptionInput");
+    var currentCar;
+
+    eventHandlers.addHandlers = function(car, inventory) {
+
```

Comment:
This might seem dumb, but bear with me on this. One of the requirement (Technical Req 3) was that this function be named `activateEvents`. In a professional setting you might get technical requirements to name things a particular way in order to work with bits of code written by someone else.